bin/addons-linter

```diff
@@ -8,13 +8,16 @@ global.localesRoot = path.join(absoluteAppRoot, 'dist', 'locale');
 global.nodeRequire = require;
 
 try {
+  // eslint-disable-next-line import/no-unresolved
```

Comment:
This is needed when running without a built addons-linter version. I think it's okay to ignore as it may just confuse people.

Reply:
Done in 67a1a4f